charts/stable/mosquitto/values.yaml

```diff
@@ -18,6 +18,9 @@ image:
 # @default -- See values.yaml
 service:
   main:
+    # When enabled, adds the `listener` option to the mosquitto config. Change
+    # this to false when using TLS.
+    addListener: true
```

Comment:
I don't think this should live under the `service.main` key. Please place this on the values root, underneath `perListenerSettings: false`.